ddl/ddl.go

```diff
@@ -544,6 +544,10 @@ func (dc *ddlCtx) newReorgCtx(jobID int64, startKey []byte, currElement *meta.El
 	return rc
 }
 
+func genBackfillJobReorgCtxID(jobID int64) int64 {
+	return -jobID
```

Comment:
too hacker here :(

Correct use of ref is the better way.

Reply:
Maybe we could remove it later.

Reply:
Yes, it should be removed in next sprint.